src/components/Header/components/NavbarDesktop.tsx

```diff
@@ -341,15 +406,43 @@ const NavBar = () => {
             })
           }
         </UlContainer>
-        <StyledLoginLink id="header-navbar-login-button" to="/login">
-          Login
-        </StyledLoginLink>
+        {testLoggedInBool
+          ? (
+            <LiSection>
+              <div style={{ margin: "0", minWidth: "150px", border: "none" }} className={clickedTitle === testData.name ? 'navTitleClicked' : 'navTitle'}>
```

Comment:
to avoid inline style as much as possible.  It is unsafe. 
https://www.iothreat.com/blog/csp-style-src-unsafe-inline.